presto-cli/src/main/java/com/facebook/presto/cli/ClientOptions.java

```diff
@@ -95,6 +121,18 @@ public ClientSession toClientSession()
                 debug);
     }
 
+    public KerberosConfig toKerberosConfig()
+    {
+        KerberosConfig config = new KerberosConfig()
+                .setConfig(new File(krb5ConfigPath))
+                .setKeytab(new File(krb5KeytabPath));
+        if (krb5CredentialCachePath == null) {
+            krb5CredentialCachePath = System.getenv("KRB5CCNAME");
```

Comment:
This is not going to work. The value of `KRBCCNAME` is something like `FILE:/var/run/ccache/krb5cc_5070`, which is not a valid path